Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ruby:2.4.1-alpine
+
+ENV RUNTIME_PACKAGES="tini" \
```

Comment:
How about trying out [virtual packages](https://github.com/gliderlabs/docker-alpine/blob/master/docs/usage.md#virtual-packages) instead of separating them via environment variables?